spec.go

```diff
@@ -80,14 +88,15 @@ func (c *SpecController) Show(ctx *app.ShowSpecContext) error {
 
 // clone does a shallow clone of the repo in the given directory and returns the "go1" or - if there
 // is no go1 branch - the "master" branch SHA.
-func clone(repo, tmpDir string) (string, error) {
-	var branch string
+func clone(repo, tmpDir, branch string) (string, error) {
 	clone := func() error {
 		gitCmd := exec.Command("git", "clone", "--depth=1", "--single-branch", "--branch", branch, repo)
 		gitCmd.Dir = tmpDir
 		return gitCmd.Run()
 	}
-	branch = "go1"
+	if branch == "" {
+		branch = "go1"
+	}
 	if err := clone(); err != nil {
```

Comment:
We should probably only degrade to using the `master` branch if a specific branch wasn't specified.